packages/ember-glimmer/lib/renderer.ts

```diff
@@ -32,7 +32,7 @@ export type IBuilder = (env: Environment, cursor: Cursor) => ElementBuilder;
 
 export class DynamicScope implements GlimmerDynamicScope {
   constructor(
-    public view: Component | {} | null,
```

Comment:
Since we removed the dynamic scope manipulation in this PR, we can revert this change